web/beacon-app/src/components/ui/SetupTenant/SetupTenant.tsx

```diff
@@ -0,0 +1,19 @@
+import { memo } from 'react';
+
+import CustomizeTenant from './CustomizeTenant';
+import TenantHeader from './TenantHeader';
+import TenantQuickStart from './TenantQuickStart';
+
+function SetupTenant() {
+  return (
+    <section className="bg-hexagon bg-contain">
```

Comment:
The Figma design includes the hexagons in the background, but they don't appear in Storybook. If the bg image should be removed, let me know.